controllers/cluster_controller.go

```diff
@@ -64,7 +64,7 @@ func (r *ClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 	// If the cluster is not found then any existing probes if present will be unregistered
 	if notFound {
 		if r.ProberMgr.Unregister(req.Name) {
-			log.Info("Cluster not found, existing prober will be removed")
+			log.Info("Cluster not found, existing prober will be removed", "namespace", req.Namespace, "name", req.Name)
```

Comment:
Doesn't the log contain these information already?

Reply:
Done